descartes_moveit/include/descartes_moveit/moveit_state_adapter.h

```diff
@@ -83,6 +83,17 @@ class MoveitStateAdapter : public descartes_core::RobotModel
     return robot_state_;
   }
 
+  /**
+   * @brief Copies the internal state of 'state' into this model. Useful for initializing the
+   *        value of joints that are not part of the active move group. Should be called after
+   *        'initialize()'.
+   */
+  void setState(const moveit::core::RobotState& state)
+  {
+    assert(static_cast<bool>(robot_state_));
```

Comment:
Would it not be better to use ROS_ASSERT_MSG in order to provide a description of the type of error?